k8s/mongodb/mongo-ss.yaml

```diff
@@ -37,14 +37,30 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mongodb
-        image: mongo:3.4.1
+        # TODO(FIXME): Do not use latest in production as it is harder to track
+        # versions during updates and rollbacks. Also, once fixed, change the
```

Comment:
Okay I will delete my comment and your response so it's easier to skim through this PR and see which comments remain to address.